lib/playlistGenerator/PlaylistGenerator.js

```diff
@@ -374,10 +384,12 @@ PlaylistGenerator.prototype.stop = function stop (){
     }
     if(isRunning.call(that)){
         that.startPromise = null;
+        that.playlistImp.expirationTime = Data.now;
```

Comment:
i'd rather you update the expirationTime every time you update the playlist
so if the process crashes/ machine killed it will automtecly be expired...